package.json

```diff
@@ -11,22 +11,27 @@
     "lint": "eslint src/*.js"
   },
   "devDependencies": {
+    "autoprefixer": "^9.6.1",
     "css-loader": "^2.1.1",
     "file-loader": "^4.0.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
-    "path": "^0.12.7",
     "mini-css-extract-plugin": "^0.6.0",
     "node-sass": "^4.12.0",
+    "path": "^0.12.7",
+    "postcss-loader": "^3.0.0",
     "sass-loader": "*",
     "style-loader": "^0.23.1",
     "webpack": "^4.32.1",
     "webpack-cli": "^3.3.2",
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
+    "bootstrap": "^4.3.1",
     "eslint": "^6.0.1",
     "fullpage.js": "^3.0.5",
+    "jquery": "^3.4.1",
```

Comment:
It seems that bootstrap has removed the `jquery` from the `peerDependencies`
https://github.com/twbs/bootstrap/blob/master/package.json#L105

Reply:
Yup you are right!
How do you know that?
Since the doc https://getbootstrap.com/ still ask developer to install `jQuery`
![image](https://user-images.githubusercontent.com/13488449/61992655-8d1b1c00-b0a4-11e9-9fd0-1ba1c9660b26.png)

Reply:
Just because I don't want to install jquery whenever it's possible. Especially you don't import it explicitly. I checked bootstrap's issues and package.json to see if the user needs to install jquery via npm package.